### ISSUE TITLE
Is source-rel-filter deprecated?

### ISSUE THREAD

**Issue by ktmn** [rosell-dk/webp-express#298]
After an update images on child sites (subdir) stopped working.

`x-webp-express-error: Sanity check failed for source (retrieved by the request_uri server var): File does not exist`

My nginx rule:

```
if ($http_accept ~* "webp") {
	rewrite ^/(.*).(jpe?g|png)$ /wp-content/plugins/webp-express/wod/webp-on-demand.php?source-rel=$request_uri&source-rel-filter=discard-parts-before-wp-content&wp-content=wp-content&%1 break;
}
```

Currently downgraded to 0.14.8 which seems to be the last version where it works for me.

Doesn't seem to be any reference to `source-rel-filter` in the source anymore, is there a new alternative I missed?

This is not urgent, have a good vocation :)

**Comment by rosell-dk** [comment]
Yes, sorry, I removed it in those desperate days where the plugin was closed due to security issues. I needed to be rough because I had no time for considering if the filter was insecure. And I don't think the feature was very widely used
I shall get back to this after my vacation, thanks :)
